src/components/ui/ConfirmDialog/ConfirmDialog.module.scss

```
@@ -0,0 +1,73 @@
+@use '@/styles/typography.scss' as typography;
+@use '@/styles/radius.scss' as radius;
+
+.overlay {
+  position: fixed;
+  inset: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: spacing.$space-24;
+  background-color: rgba(0, 0, 0, 0.5);
+  // FixedButton 등 page 내 position: fixed 요소들보다 위에 떠야 함.
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)